src/components/views/rooms/RoomSublist2.tsx

```diff
@@ -363,9 +370,9 @@ export default class RoomSublist2 extends React.Component<IProps, IState> {
             // floats above the resize handle, if we have one present. If the user has all
             // tiles visible, it becomes 'show less'.
             let showNButton = null;
-            if (tiles.length > nVisible) {
+            if (this.numTiles > this.numVisibleTiles) {
                 // we have a cutoff condition - add the button to show all
-                const numMissing = tiles.length - visibleTiles.length;
+                const numMissing = this.numTiles - visibleTiles.length;
```

Comment:
Hmm, is there a reason to use`visibleTiles.length` in some parts and `this.numVisibleTiles` in others...? They should be the same right? If so, it would be easier to read if we pick one way and stick with it.

Reply:
It should be consistent within the functions, hopefully. We save nanoseconds by using the length.

Reply:
oh god it's not consistent. Will fix.